spm-monitor-utils/src/main/java/com/sematext/spm/client/config/DataConfig.java

```diff
@@ -27,6 +29,7 @@
   private String dbPassword;
   private String dbAdditionalConnectionParams;
   private boolean dbVerticalModel = false;
+  private List<DbDriverConfig> dbDriver;
```

Comment:
Thinking if there is better naming for this... We now have Driver object with `url` and `driverClass`. Problem is with `url` part. Name can't be Connection either because `username` and `password` are configured separately. Don't have a good suggestion, if we don't think of anything better, let's keep.

Reply:
Even I thought about a better name, but cannot find one. So went with this naming.